src/mol-model/structure/model/properties/utils/secondary-structure.ts

```diff
@@ -183,10 +460,10 @@ function calcBackboneHbonds(hierarchy: AtomicHierarchy, conformation: AtomicConf
         position(cAtom, cPos)
         position(caAtom, caPos)
 
-        const { indices, count, squaredDistances } = lookup3d.find(caPos[0], caPos[1], caPos[2], caMaxDist)
+        const { indices, count } = lookup3d.find(caPos[0], caPos[1], caPos[2], caMaxDist)
 
         for (let j = 0; j < count; ++j) {
-            if (squaredDistances[j] < caMinDistSq) continue
+            // if (squaredDistances[j] < caMinDistSq) continue
```

Comment:
Why not do this test?

Reply:
Does not change output, does not decrease computation time. Omitted for simplicity.